### ISSUE TITLE
Installation Fails Due to issue #11160

### ISSUE THREAD

**Issue by KirinDave** [syl20bnr/spacemacs#11289]
Attempting to follow instructions from the website with a fresh install of emacs on ubuntu 18 leads to the following error message: 

````
Warning (initialization): An error occurred while loading '/home/𐑕𐑳𐑥/.emacs.d/init.el':

error: Package 'dash-functional-1.2.0' is a system package, not deleting

To ensure normal operation, you should investigate and remove the cause of the error in your initialization file...
````

This isn't trivial for an end user to fix.

**Comment by Miciah** [comment]
You reference issue #11160, and it looks like you are reporting the same issue.  Why did you open a new issue instead of commenting on the existing issue? (Please don't take that the wrong way—it is important the problem be resolved, but I'm confused as to why you opened a new issue.)

**Comment by KirinDave** [comment]
I'll close it and repost, but I reported it as separate because that issue is about the underlying mechanical problem and does not represent that installations are failing. When I spoke with the spacemacs twitter account yesterday they seemed surprised that builds were broken despite the fact that the bug is not especially new, so I decided to call that out directly since install being broken is more severe.

Sorry!
